DRAFT_COROLLARY.md

```diff
@@ -269,8 +266,8 @@ A. _Constraints_: `is_base_of_v<T, std::remove_cvref_t<U>>` is `true`.
 
 B. _Mandates_: `T` is a complete type.
 
-C. _Effects_: Equivalent to `polymorphic(std::allocator_arg_t{}, A{},
-   std::in_place_type_t<std::remove_cvref_t<U>>{}, std::forward<U>(u))`.
+C. _Effects_: Equivalent to `polymorphic(allocator_arg, A{},
+   in_place_type<std::remove_cvref_t<U>>, std::forward<U>(u))`.
```

Comment:
Remove std:: on remove_cvref_t (keep it on std::forward, because of LWG policy that nobody really likes any more!)

Reply:
I will raise a new PR to address this.  Thanks for catching, I missed this.